stdlib/public/SDK/AppKit/AppKit.swift

```diff
@@ -78,9 +78,8 @@ struct _NSViewMirror : _MirrorType {
       // This code checks that we aren't trying to log the same view recursively - and if so just returns
       // nil, which is probably a safer option than crashing
       // FIXME: is there a way to say "cacheDisplayInRect butDoNotRedrawEvenIfISaidSo"?
-      switch _NSViewMirror._views.member(_v) {
-        case nil:
-          _NSViewMirror._views.addObject(_v)
+      if !_NSViewMirror._views.contains(_v) {
+          _NSViewMirror._views.insert(_v)
```

Comment:
Could you fix up the indentation after removing one level of nesting?  We use two spaces.